packages/cubejs-cockroach-driver/test/BaseDBRunner.ts

```diff
@@ -0,0 +1,114 @@
+import { PreAggregationPartitionRangeLoader } from '@cubejs-backend/query-orchestrator';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { StartedDockerComposeEnvironment } from 'testcontainers';
+
+export class BaseDbRunner {
```

Comment:
Generally speaking, you should move this class to the `@cubejs-backend/testing-shared` package and import it here.